### DISCUSSION TITLE
Welcome to Chakra UI Vue Next Discussions! 🎉

### DISCUSSION THREAD

**Discussion by codebender828** [chakra-ui/chakra-ui-vue-next#45] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wensonsmith** [comment]
I want to known when will the first release be ?

thank you

**Comment by feitian124** [comment]
最近学习了 tailwind css，一个很棒的 css 工具类框架。然后找到了 chakraui，现在对 vue3 版本很有兴趣。
昨天看了你在 vue conf 杭州 2021 的演讲，很棒的演讲，也进一步加深了我的兴趣。
本来想自己封装一个 tailwind vue3版本，既然这里已经有一个不错的项目了，想参与进来。
我不是专业的前端，但是 vue，js 基础还不错。
请问怎么参与？很多工作是不是只是将 chakra 的 react 版本移植到 vue 版本即可？
